### DISCUSSION TITLE
domain

### DISCUSSION THREAD

**Discussion by alonelove7** [vpnhood/VpnHood#150] [Q&A]
Hello
it's possible to use domain instead of IP?

**Comment by trudyhood** [comment]
Are you using AccessServer or StandAlone server?
In the StandAlone server you can use a valid certificate and using its domain instead of ip.

**Reply by alonelove7** [reply]
yes i use my own server
Can you share the desired configuration with me? , please
A domain is needed for more stability
For use in my channel

**Reply by trudyhood** [reply]
Oops! It looks like the command line for the code generator does not support it. Please create an issue, and we will fix it later. I assume it needs a little work because we have already implemented it in the client and server.

**Reply by alonelove7** [reply]
Sure 
thank you
